CMakeLists.txt

```diff
@@ -117,6 +120,22 @@ add_psi4_plugin(v2rdm_casscf
     focas_transform_teints.F90
 )
 
+#==================================================================
+if(${ENABLE_OPENRDM})
+   string(COMPARE EQUAL "${openrdm_DIR}" "" _bool_result)
+   if(${_bool_result})
+      message(FATAL_ERROR "${Red}Please set the \"openrdm_DIR\" variable to .../build/stage/share/cmake/openrdm${ColourReset}")
```

Comment:
`openrdm_DIR` is _a_ way but not the only way to communicate location to `find_package`. I'm wary of requiring it, especially with advice to set it to such a custom spot. If you run `find_package`, and it can't, it'll print a message saying set `openrdm_DIR` (among other advice). Does that address your concern?

Reply:
The error message in line 127 and directions provided in documentation for OpenRDM should be sufficient for the user to notice the host directory for the **openrdmConfig.cmake** file which _find_package(CONFIG)_ function requires. The reason for adding configure script including the specific address was only to indicate to Eugene an example where this variable was set on my computer with the intention that it should be deleted afterwards as this specific script and the address within does not serve for the general purpose of a potential user and you are right in that sense. I could send him this file via Slack.